config/admissions.uiowa.edu/config_split.config_split.admissions.yml

```diff
@@ -13,9 +13,11 @@ theme: {  }
 blacklist: {  }
 graylist:
   - config_split.config_split.admissions
+  - field.field.node.person.field_person_territory
```

Comment:
Just as an FYI, it is unnecessary to put the field instance config in here because having the field storage will cause it to be pulled in as a dependency.